### PR TITLE
 Add KFC to Wipeall, Prevent Privilege Escalation 

### PR DESCRIPTION
**Pull request by lkse**
Add KFC to wipeall function
update the self-privilege escalation function to include users that are not the user
also snitch on them via the RAPlog function

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -1016,6 +1016,13 @@ def ApplyUserEdit(form, session):
             return
         OriginalPriv = OriginalPriv[0][0]
         if int(Privilege) > OriginalPriv:
+            RAPLog(session f"Attempted to set their own privilege to {Privilege}: (Original: {OriginalPriv})") # Snitch on them
```
Comment (7ez) [inline functions.py]: u cant even edit urself???

Comment (7ez) [reply functions.py]: 💀 most useless feature of 2022

Comment (lkse) [reply functions.py]: you can't edit yourself but you can give other people privileges higher than your own. this prevents that

Comment (7ez) [reply functions.py]: You can edit yourself but not give yourself higher privileges

Comment (7ez) [reply functions.py]: and there's no point in "snitching"

**Comment by lkse** [comment]
Added comments and fixed the raplog push

**Comment by 7ez** [comment]
No brah

**Comment by 7ez** [comment]
U make the most useless shit ever

**Comment by Xyrohh** [comment]
shut up aochi

**Comment by 7ez** [comment]
shut up tranny

**Pull request closed without merging**